src/Ionide.ProjInfo/Library.fs

```diff
@@ -1056,7 +1107,19 @@ type WorkspaceLoaderViaProjectGraph private (toolsPath, ?globalProperties: (stri
                     // and tell the graph to use all as potentially an entrypoint
                     let nodes =
                         g.ProjectNodes
-                        |> Seq.map (fun pn -> ProjectGraphEntryPoint pn.ProjectInstance.FullPath)
+                        |> Seq.choose (fun pn ->
+                            match pn.ProjectInstance.GetProperty("IsTraversal") with
+                            | null ->
+                                ProjectGraphEntryPoint pn.ProjectInstance.FullPath
+                                |> Some
+                            | p ->
+                                match bool.TryParse(p.EvaluatedValue) with
+                                | true, true -> None
+                                | true, false ->
+                                    ProjectGraphEntryPoint pn.ProjectInstance.FullPath
+                                    |> Some
+                                | false, _ -> None
+                        )
```

Comment:
Here's the main impact for the graph build loader - if any of the project nodes are traversal projects, editors won't really know what to do with them so just...don't report them.